modules/apigee_m10n_teams/src/Entity/TeamsPurchasedPlan.php

```diff
@@ -113,25 +114,30 @@ public function getTermsAndConditions(): bool {
     }
   }
 
-  /**
-   * {@inheritdoc}
-   */
-  public function getTeam() {
```

Comment:
Since this is public it should be deprecated instead of removed. It is also exposed as a field in `/config/apigee-monetization/purchased-plan-settings/display`. I think this is the only method that references `TeamsPurchasedPlan::getTeamReference()` so that code could just be folded in.

Reply:
I added back this method so that it continues to be exposed as a field in the display settings page.